src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -186,7 +187,14 @@ private void ShowHidePopup(ShowHideFlags flag, Popup popup)
                     break;
                 case ShowHideFlags.Show:
                     // Show InCanvas search just in case, when mouse is over workspace.
-                    popup.IsOpen = DynamoModel.IsTestMode || IsMouseOver;
+                    var displayPopup = DynamoModel.IsTestMode || IsMouseOver;
+                    if (displayPopup && popup == NodeAutoCompleteSearchBar)
+                    {
+                        ViewModel.NodeAutoCompleteSearchViewModel.PortViewModel.SetupNodeAutocompleteWindowPlacement(popup);
+                    }
+                    popup.IsOpen = displayPopup;
+                    popup.CustomPopupPlacementCallback = null;
```

Comment:
Is this a must have or just sanity cleanup?

Reply:
Yes, otherwise the `CustomPopupPlacementCallback` continues pointing to `PlaceAutocompletePopup`, which can be invoked unexpectedly, say when in-canvas search popup is opened.